tests/unit/ext/switches/brocade.py

```diff
@@ -234,8 +234,8 @@ def test_apply_networking(self, switch, nic, network):
 
         # Test action to add a vlan
         action_vlan = model.NetworkingAction(nic=nic,
-                                         new_network=network,
-                                         channel='vlan/102')
+                                             new_network=network,  # noqa
+                                             channel='vlan/102')  # noqa
```

Comment:
You shouldn't need it here either. `--pep8` will accept stuff like:

```python

def foo_bar(x, y, z, w):
    pass


foo_bar(
    1,
    2,
    3,
    4,
)

```

So you can avoid running afoul of the line-length rules by putting the `nic` arg on the next line and having it not indented nearly as far.